public/formEnumLookups.js

```diff
@@ -1,79 +1,66 @@
 export const FILTER_ITEMS = {
   'n95s': {
-    name: 'ftm-accept-item-n95s',
-    isSet: false
+    name: 'ftm-accept-item-n95s'
   },
   'surgical masks': {
-    name: 'ftm-accept-item-surgical-masks',
-    isSet: false
+    name: 'ftm-accept-item-surgical-masks'
+  },
+  'handmade masks': {
+    name: 'ftm-accept-item-handmade-masks'
+  },
+  'homemade masks': {
+    name: 'ftm-accept-item-homemade-masks'
   },
   'homemade masks (specify type in other)': {
-    name: 'ftm-accept-item-homemade-masks',
-    isSet: false
+    name: 'ftm-accept-item-homemade-masks-specify-type'
   },
   'face shields': {
-    name: 'ftm-accept-item-face-shields',
-    isSet: false
+    name: 'ftm-accept-item-face-shields'
+  },
+  'face shields - 3d-printed/makerspace': {
+    name: 'ftm-accept-item-face-shields-makerspace'
   },
-  'faceshields - 3d-printed or makerspace (specify type in other)': {
-    name: 'ftm-accept-item-face-shields-makerspace',
-    isSet: false
+  'face shields - 3d-printed or makerspace (specify type in other)': {
```

Comment:
Do you think that the stuff in parens is actually necessary here? I think it makes sense on the google form but for a filter I think it's weird.

Not a blocker and could just be a follow up

Reply:
I agree, but that's how it's coming in from the forms -- in the US we have the parenthetical and in Canada we don't. Kind of annoying but I was told that it'd be more work to back-fit the data in the sheets than just to double up here.

I could change the translation so that they look the same though in `en.json` -- since `(specify type in other)` makes no sense in a filter

Reply:
yeah I think that updating the translation would be great

Reply:
@nburt I reworked so that they just use the same translation as the one *without* "(specify type in other")